packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.8.4+4
+
+* Support adding WebP for iOS
```

Comment:
```suggestion
* iOS: allow picking images with WebP format.
```

Reply:
Nit: should be "Allows", not "Allow", in the suggested change.